### PR TITLE
PWA notice for unsupported browser

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: locales/en/messages.json [hunk 1]
```diff
@@ -51,6 +50,10 @@
         "message": "You can install it as an standalone application on your device if you want. Search for the option in your browser address bar or menu.",
         "description": "Text of the window that appears when the app is ready to be installed and used offline"
     },
+    "pwaNonCompatibleBrowser": {
+        "message": "Betaflight app requires chrome, chromium, edge or vivaldi browser.",
```
Comment (nerdCopter) [inline locales/en/messages.json:54]: suggest capital-case browser names.

**Review by nerdCopter — APPROVED** [review]

**Comment by nerdCopter** [comment]
delightful
image

**Comment by HThuren** [comment]
a small issue
image

**Comment by nerdCopter** [comment]
so `il8n` not loaded at that juncture, or just not working proper?

**Comment by haslinghuis** [comment]
i18n.getMessage does not translate at this point :upside_down_face:

**Comment by HThuren** [comment]
> i18n.getMessage does not translate at this point 🙃

Can we make it to do translate, ex explicit call or ... ?

**Comment by nerdCopter** [comment]
i feel like this should be approved/merged and language correction could come later.  Such would stop us from having to tell users when we give them the link is Discord.

**Comment by HThuren** [comment]
> i feel like this should be approved/merged and language correction could come later. Such would stop us from having to tell users when we give them the link is Discord.

Agree, we can try to work out a solution another time.

**Review by HThuren — APPROVED** [review]

**Pull request merged**